README.md

```diff
@@ -1,2 +1,36 @@
 # rca-barefoot-team6-bn
+
+# Barefoot
+
+Barefoot is a project which helps people book accomodation in places that they plan to stay at. 
+The application is mostly for tourists and generally anyone who wants to know and book accomodation facilities in an area when they plan to visit. 
+
+
+
+
+
+## Installation
+
+- Clone the project 
+
+- Install dependencies with
+```bash 
+npm install
+```
+- Create a ```bash .env``` file to the root directory of the project 
+- Copy content from ```bash .env.example``` to the ``` .env``` file
+- To each key, assign its corresponding value which will be used while running the project
+
+- Start the server using
+```bash
+npm run dev
+```
```

Comment:
This is for development only; what about build, test and start production server ?